packages/components/tests/dummy/app/templates/components/form/textarea.hbs

```diff
@@ -199,9 +199,428 @@
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Form::Textarea::Field placeholder="Workspace description" \{{on "blur" myAction}} as |F|>
+        <F.Label>Email</F.Label>
+      </Hds::Form::Textarea::Field>
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Form::Textarea::Field placeholder="Workspace description" {{on "blur" this.noop}} as |F|>
+    <F.Label>Email</F.Label>
+  </Hds::Form::Textarea::Field>
+  <p class="dummy-paragraph">You can use different events, depending on your context/need (eg.
+    <code class="dummy-code">input</code>,
+    <code class="dummy-code">blur</code>,
+    <code class="dummy-code">change</code>).</p>
+
+  <h5 class="dummy-h5">Custom width</h5>
+  <p class="dummy-paragraph">By default the textarea control width is set to fill the parent container. It's possible to
+    pass a custom width for the control using the
+    <code class="dummy-code">@width</code>
+    argument:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Form::Textarea::Field @width="200px" as |F|>
+        <F.Label>Short description</F.Label>
+      </Hds::Form::Textarea::Field>
+    '
+  />
+  {{! template-lint-enable no-unbalanced-curlies }}
+  {{! prettier-ignore-end }}
+  <p class="dummy-paragraph">Renders to:</p>
+  <Hds::Form::Textarea::Field @width="200px" as |F|>
+    <F.Label>Short description</F.Label>
+  </Hds::Form::Textarea::Field>
+
+  <h4 class="dummy-h4">Form::Textarea::Base</h4>
+
+  <p class="dummy-paragraph">
+    As mentioned above, the "base" element is intended
+    <strong>only</strong>
+    for those rare cases where the "field" variant can't be used, and a custom implementation needs to be done. For this
+    reason we will not go too much in detail on how to use it: most of the explanations above apply also to the "base"
+    variant of the component, and for further details refer to the
+    <a href="#component-api">Component API</a>
+    section on this page, or speak with one of the design system team members.
+  </p>
+  <p class="dummy-banner dummy-banner--info dummy-paragraph"><FlightIcon @name="info" />
+    Notice: when the "base" textarea is used, the developer is completely responsible for the correct implementation of
+    the form control, including its accessibility conformance.</p>
+  <p class="dummy-paragraph">To give just an example, this could be an invocation of the "base" component you would use:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
+    @language="markup"
+    @code='
+      <Hds::Form::Textarea::Base
+        @value="My workspace"
+        placeholder="Workspace description"
+        @isRequired=\{{true}}
+        \{{on "blur" myAction}}
+      />
```

Comment:
```suggestion
      <Hds::Form::Textarea::Base
        @value="My workspace"
        aria-label="Short description"
        placeholder="Workspace description"
        @isRequired=\{{true}}
        \{{on "blur" myAction}}
      />
```

For the base examples, considering they can easily be copied and adopted in product codebases, I'd be tempted to add an `aria-label` as an equivalent for label.